latex/DPIA/x.log

```diff
@@ -0,0 +1,85 @@
+This is LuaTeX, Version 1.10.0 (TeX Live 2019/Arch Linux)  (format=lualatex 2020.1.13)  30 JAN 2020 18:55
```

Comment:
I don't think this file should be included.

Reply:
Agreed. Will be removed with the next commit.